packages/chart/data/api/prod-env.yaml

```diff
@@ -0,0 +1,19 @@
+image:
+  repository: alvnoas/alvtime-web-api
+
+config:
+  AdminGroup: "d2bff530-4037-4a53-823e-a29dac3905ef"
+  ASPNETCORE_ENVIRONMENT: "Production"
+  AzureAd__ClientId: "3644a03b-f801-4b67-b487-3353684fe7fc"
+  AzureAd__Domain: "api://3644a03b-f801-4b67-b487-3353684fe7fc"
+  AzureAd__Instance: "https://login.microsoftonline.com/"
+  AzureAd__IssuerId: "https://login.microsoftonline.com/76749190-4427-4b08-a3e4-161767dd1b73"
+  AzureAd__TenantId: "76749190-4427-4b08-a3e4-161767dd1b73"
+  TimeEntryOptions__FlexTask: "55"
```

Comment:
Er det sånn at disse overskriver variablene i appsettings.json? Og blir da denne filen man oppdaterer noe hvis noen av disse variablene endrer seg, eller man skal legge til nye? Disse blir jo også satt som environment variabler gjennom pipeline, men det skal kanskje fjernes?

Reply:
Ja disse overskriver variablene i appsettings.json ved at de ender opp som environment variabler i containeren når den starter i kubernetes. Så ja denne må oppdateres dersom en av de endres for prod. Ja alle deployment pipelinene til både test og prod skal erstattes av deploy-pipeline.yaml som ligger på rot.

Reply:
Så dvs at man må lage en ny commit, få review og release ny branch hver gang man skal oppdatere en environment variabel da? Høres litt mer arbeidsomt ut enn å bare endre på det i pipeline variablene.

Reply:
Enig. Vi kan godt flytte den/de til å bli satt i pipeline. Jeg kan gå igjennom de og finne ut om det er noen flere av de som kan kalkuleres utifra infrastruktur bygget også.